src/db/index.ts

```diff
@@ -1,20 +1,11 @@
 import { drizzle } from "drizzle-orm/postgres-js";
 import postgres from "postgres";
 
-const setup = () => {
```

Comment:
I noticed this was causing some funny type issues on the API response so I made a bit of a change here. Previously this was causing the response type to be `never[] | {...}` which meant I'd have to cast my response type wherever it was used. I think this change should be safe.